### DISCUSSION TITLE
Pokeclicker 10.2 update - breaking scripts lategame

### DISCUSSION THREAD

**Discussion by ssiatkowski** [Farigh/pokeclicker-automation#203] [General]
On my main save in Galar the only option showing up now is "Auto attack". However, I also have an AC run that's in Hoenn where everything still works. I assume some later functionality is breaking the script.

By the way haven't had a chance to say this but you've done some amazing work Farigh!!

**Comment by Farigh** [comment]
This should now be fixed (by #200)
